src/app/components/app-build-item/app-build-item.component.html

```diff
@@ -12,8 +12,15 @@
     <span *ngIf="tag" class="bold larger">{{ tag }}</span>
   </div>
   <div class="column is-2">
-    <span class="larger" *ngIf="build?.repository?.full_name">{{ build?.repository?.full_name }}</span>
-    <span class="larger" *ngIf="!build?.repository?.full_name">{{ build?.data?.repository?.full_name }}</span>
+    <div class="repo-branch-column">
+      <span class="larger bold" *ngIf="build?.repository?.full_name">{{ build?.repository?.full_name }}</span>
+      <span class="larger bold" *ngIf="!build?.repository?.full_name">{{ build?.data?.repository?.full_name }}</span>
```

Comment:
Really a minor remark, but I see sometimes this underscore_separated variables and sometimes camelCase ones. We should probably stick to one convention.

Reply:
underscore separated values comes from database. we have strict tslint configuration that doesn't allow this inside src/**/*.ts code.